### ISSUE TITLE
ADMIN : News : Outdated-RSVP-link 404 intentional?

### ISSUE THREAD

**Issue by sarmaswe** [eastbaydsa/website#160]
URLs: 
https://www.eastbaydsa.org/news/2018/6/4/tdu-fundraiser
https://www.eastbaydsa.org/event-social-2018-06-09-teamsters-democratic-union-fundraiser

I think it would be totally fine for this to stay as is, as the event is in the past -- and I remember some conversation about this. I just wanted to double check that this is as expected. Is that correct, @wesholing?

![screen shot 2018-07-07 at 9 51 16 pm](https://user-images.githubusercontent.com/13740788/42416920-f01a1ee0-822f-11e8-86eb-4d0f10879085.png)
![screen shot 2018-07-07 at 9 51 25 pm](https://user-images.githubusercontent.com/13740788/42416921-f2c41cfe-822f-11e8-8c2e-296970116445.png)

**Comment by sarmaswe** [comment]
@wesholing this is as expected, right?

**Comment by wesholing** [comment]
Yeah, I think the link is fine to leave, but I've updated it to the past event URL (which changed as a result of the content migration). The blog post should be good now.
